scripts/timestamp.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
+const semver = require('semver');
+const dayjs = require('dayjs');
```

Comment:
Sorry, but another change will be needed for this: to remove `moment-mini` from the dependencies and add `dayjs` in the `package.json`.

Reply:
Sure! I will change this too

Reply:
Great! you can find the corresponding `dayjs` package version again in the concerto `package.json`

Reply:
Ohh sorry for that I had installed latest version is that okay?
or should I install one that is there in concerto's `package.json`

Reply:
@jeromesimeon I had added the corresponding version of `dayjs`